sourcegraph/sourcegraph
# Insights/fix-series-hover-states

Closes https://github.com/sourcegraph/sourcegraph/issues/36462
Closes https://github.com/sourcegraph/sourcegraph/issues/36325

## Test plan

- Open storybook
- Go to Smart View Grid
- Find any chart with multiple series
- Hover over legend item should dim all but that legend item
- Hover over series should dim all other series

## App preview:

- [Web](https://sg-web-insights-fix-series-hover-states.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-wbieyvlvqb.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
